samples/1.0.0-rc1/HelloMvc/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:14.04
```

Comment:
The rc1 Dockerfile should assume the existence of the Microsoft rc1 image. So just from Microsoft/aspnet:1.0.0-rc1, copy app, restore, build. Like the others.